src/views/Home.jsx

```diff
@@ -43,32 +47,52 @@ export function Home({ listToken, setListToken }) {
 			}
 		});
 	};
+
+	function openForm() {
+		setDisplayForm(true);
+	}
+
+	function closeForm() {
+		setDisplayForm(false);
+	}
+
 	return (
-		<div className="Home">
-			<p>
-				Hello from the home (<code>/</code>) page!
-			</p>
-
-			<div>
-				<button onClick={createToken}>Create a new list</button>
-
-				<form onSubmit={handleSubmit}>
-					<label htmlFor="existingToken">
-						Join an existing list
-						<input
-							type="text"
-							id="existingToken"
-							value={existingToken}
-							onChange={handleChange}
-							required
-						/>
-					</label>
-
-					<button type="submit">Submit</button>
-				</form>
-
-				{errorMessage && <p>{errorMessage}</p>}
+		<div className="h-screen flex flex-col items-center justify-around pt-8">
+			<div className="flex flex-col items-center gap-6">
+				<img src="/img/logo.svg" alt="Shroomy logo" className="w-20 mt-6" />
+				<h1 className="text-6xl font-logo -my-2">Shroomy</h1>
+				<p className="text-2xl font-logo">your groovy shopping companion</p>
+
+				<div className="flex flex-col items-center gap-6 mt-6">
+					<button
+						onClick={createToken}
+						className="bg-main-darkest text-white rounded-3xl shadow-lg py-2 px-12 hover:bg-white hover:text-main-darkest hover:border-main-darkest hover:border-[1.5px]"
+					>
+						Create a new list
+					</button>
+
+					<button
+						className="bg-white text-main-darkest border-[1.5px] border-main-darkest rounded-3xl shadow-lg py-2 px-12 hover:bg-main-darkest hover:text-white"
```

Comment:
@neshacascia  adding hover:scale-105 and the shadow effect for consistency.

```suggestion
						className="bg-white text-main-darkest border-[1.5px] border-main-darkest rounded-3xl shadow-[0_4px_4px_rgba(0,0,0,0.4)] py-2 px-12 hover:bg-main-darkest hover:text-white hover:scale-105"
```

Reply:
Thanks for the suggestions! Not going to add `hover:scale-105` as I noticed with the last call to action, upon clicking, the button would stick on top of the `FormModal` until the mouse moved outside of the button. Adding the borders was enough to prevent the layout from shifting!